api/report_server.thrift

```diff
@@ -363,12 +363,12 @@ service codeCheckerDBAccess {
                  throws (1: shared.RequestFailed requestError),
 
   NeedFileResult needFileContent(
-                                 1: i64 run_id,
-                                 2: string filepath)
+                                 1: string filepath,
```

Comment:
Add documentation to the API function.

If matching filepath, content_hash is found in the DB return the correspondig,existing NeedFileResult.fileId,  NeedFileResult.needed=false
* If only the content_hash matches, a new a new NeedFileResult.fileId is returned,  NeedFileResult.needed=false
* If the filepath matches, but content_hash not, a new file_id is generated and NeedFileResult.needed=true.
* If none of the conent_hash and the filepath matches a new file_id is generated and NeedFileResult.needed is false in the response.